org.eclipse.gef/.settings/.api_filters

```diff
@@ -64,6 +64,14 @@
             </message_arguments>
         </filter>
     </resource>
+    <resource path="src/org/eclipse/gef/EditPartViewer.java" type="org.eclipse.gef.EditPartViewer">
+        <filter id="404000815">
+            <message_arguments>
+                <message_argument value="org.eclipse.gef.EditPartViewer"/>
+                <message_argument value="getEditPartForModel(Object)"/>
```

Comment:
Based on the documentation, the EditPartViewer interface shouldn't be implemented by clients. I wonder if one can instead add `@noimplement` and avoid this entry?

Reply:
I fear still we need that. The reason for the problem is that there could be children that have a method with the same name and we would break these children.

Reply:
Fair point.